src/javascript/EditPanel/EditPanelContent/FormBuilder/Section/FieldSet/Field/SelectorTypes/Picker/PickerDialog/Views/List/List.spec.jsx

```diff
@@ -152,6 +155,25 @@ describe('PickerDialog - List view', () => {
         expect(cmp.props().data[0].name).toContain('Home');
     });
 
+    it('should have non selectable content', () => {
+        setQueryResult({
+            jcr: {
+                result: queryResultWithChildren
+            }
+        });
+        defaultProps.pickerConfig.showOnlyNodesWithTemplates = true;
+
+        const cmp = shallowWithTheme(
+            <List {...defaultProps}/>,
+            {},
+            dsGenericTheme
+        )
+            .find('ContentTable');
+
+        expect(cmp.props().data[0].selectable).toBeFalsy();
```

Comment:
Can not be null so you can use .toBe(false) and toBe(true)

Reply:
Thank you for the hint!